pylabrobot/machine.py

```diff
@@ -56,11 +58,18 @@ async def stop(self):
 
   def __del__(self):
     if self.setup_finished:
-      self.stop()
+      self.backend.stop()
```

Comment:
I think we want to call `self.stop` here in case a frontend has a state (actually `Machine` already does: `_setup_finished`)